_posts/2017-10-30-lambda-world-conference.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: post
+asset-type: post
+name: Lamda-World-Conference-2017
+title: 'Lambda World Conference 2017'
+date: 2017-10-30 12:00:00 +00:00
+author: Raquel M Carmena
+image:
+  src: /assets/img/custom/blog/2017-10-30-lambda-world-conference.jpg
+tags:
+- functional programming
+- category theory
+- lambda calculus
+- conference
+
+---
+Thanks Codurance for giving me the opportunity of attending <a href="http://www.lambda.world/" target="_blank">**Lambda World Conference 2017**</a> last 26th and 27th October in Cádiz, the largest _Functional Programming_ event in Spain consisting of two intense days:
```

Comment:
is it the largest?

Reply:
In Spain, but it seems I'm selling ;-) I'm going to remove it.

Reply:
Personally, I don't have a problem with that, it's just that I wasn't sure if it was really the case